catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/PdfInputTransformer.java

```diff
@@ -175,18 +204,19 @@ private MetacardImpl initializeMetacard(String id, String contentInput) {
                     .flatMap(Collection::stream)
                     .collect(Collectors.toSet());
 
-            metacard = new MetacardImpl(new MetacardTypeImpl(BasicTypes.BASIC_METACARD.getName(),
-                    BasicTypes.BASIC_METACARD,
+            metacard = new MetacardImpl(new MetacardTypeImpl(metacardType.getName(),
```

Comment:
Can't speak for @jlcsmith , but I'd suggest the rule of thumb should be "If it doesn't facilitate discovery, it's not worth mapping. If it does facilitate discovery, we need to talk about why it's not in the normalized taxonomy." There are certainly attributes which only have value for one specific format that might not be worth adding to the taxonomy, but that's usually figured out on a case by case basis.